### PR TITLE
test(timepicker): migrate to testbed

### PR DESCRIPTION
**Pull request by sendilkumarn**
part of #563

### CODE REVIEW

File: src/timepicker/timepicker.spec.ts [hunk 1]
```diff
@@ -45,722 +45,699 @@ function expectToDisplayTime(el: HTMLElement, time: string) {
...
-           });
-         });
-       })));
+    it('should update inputs value on model change', () => {
+      const html = `<ngb-timepicker [ngModel]="model"></ngb-timepicker>`;
+
+      const fixture = createTestComponent(html);
+      fixture.componentInstance.model = {hour: 13, minute: 30};
+      fixture.detectChanges();
+      fixture.whenStable().then(() => {
```
Comment (jnizet) [inline src/timepicker/timepicker.spec.ts]: This pyramid of callbacks could be flattened with promise chaining. I did it in the typeahead test. That makes things easier to read IMHO, and reduces the indentation level.

```
    fixture.whenStable().then(() => {
        ...
        return fixture.whenStable();
    }).then(() => {
        ...
        return fixture.whenStable();
    }).then(() => {
        ...
    });
```

Comment (pkozlowski-opensource) [inline src/timepicker/timepicker.spec.ts]: Yep. But let's do it in a separate PR. It is not directly related to `TestBed`
